roles/sensu-check/tasks/monitoring.yml

```diff
@@ -0,0 +1,34 @@
+---
+- block:
+  - name: "remove {{ sensu_check_role }} role related unwanted sensu checks"
+    sensu_check_dict:
+      name: "{{ item.key }}"
+      check: "{{ item.value }}"
+      state: absent
+    when: item.value.condition | default('true') | bool
+    with_dict: "{{ sensu_removed[sensu_check_role]|default({}) }}"
+    notify: restart sensu-client missing ok
+  when: sensu_removed[sensu_check_role] is defined
+
+- block:
+  - name: "install {{ sensu_check_role }} role related sensu checks"
```

Comment:
does this variable interpolation in the task name actually work?  it used to not and would literally print `{{ variable }}` rather than expanding it out.